.ci/jobs/apm-server-mbp.yml

```diff
@@ -30,8 +30,8 @@
           after: true
           before: true
         prune: true
-        shallow-clone: true
-        depth: 3
+        shallow-clone: false
+        depth: 100
```

Comment:
Let's grab the last 100 commits which should be sufficient for the time being.

Reply:
this should only affect the default checkout